packages/database/prisma/schema.prisma

```diff
@@ -197,13 +198,15 @@ model Purchase {
   state                String?
   country              String?
   couponId             String?
+  bulkCouponId         String?
```

Comment:
This is part of Step 1 ([EaC](https://www.prisma.io/dataguide/types/relational/expand-and-contract-pattern)), expanding the schema. The schema is ready to merge on Planetscale as Deploy Requests as well.